### PR TITLE
Adding additional options to modal widget.

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) updates the `modal`'s options list with some additional useful options.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_modal.html
- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_modal.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/7aa94564d85e408baea01abc5315a0441401c375/app/code/Magento/Ui/view/base/web/js/modal/modal.js#L47

whatsnew
Added multiple missing options to the [Modal widget](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_modal.html) in the _JavaScript Developer Guide_.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.2/javascript-dev-guide/widgets/widget_modal.md [hunk 1]
```diff
@@ -152,6 +176,41 @@ Sets a margin between slide modal windows.
 
 **Default value**: 45
 
+### `modalSubTitle` {#modal_modalSubTitle}
+The modal subtitle element selector.
```
Comment (rogyar) [inline src/guides/v2.2/javascript-dev-guide/widgets/widget_modal.md]: Could I kindly ask you to provide more information about the subtitle? Particularly, what is the difference between title and subtitle?

 Thank you!

Comment (eduard13) [reply src/guides/v2.2/javascript-dev-guide/widgets/widget_modal.md]: Actually the purpose of  `subTitle` was described for another option were we set the text for subTitle:  `subTitle`.
https://github.com/magento/devdocs/pull/6118/files#diff-ee454d3c5298af295b7f3eef90f0da47R229
Please let me know if we need to adjust it somehow.

Thank you.

Comment (rogyar) [reply src/guides/v2.2/javascript-dev-guide/widgets/widget_modal.md]: Got it, missed that point. Thank you

**Pull request merged**